chalice/Makefile

```diff
@@ -30,4 +30,3 @@ deploy: clean build
 	@echo -e "#########################################################\n"
 	./build_deploy_config.sh $(DEPLOYMENT_STAGE)
 	chalice deploy --no-autogen-policy --stage $(DEPLOYMENT_STAGE) --api-gateway-stage $(DEPLOYMENT_STAGE)
-	rm -rf chalicelib
```

Comment:
What's this bit about?

Reply:
I added this awhile ago to keep the working dir clean, but removing it now to preserve `chalicelib/index.html` (same pattern as other repos)